internal/ccsmp/lib/include/solclient/solClientDeprecated.h

```diff
@@ -34,7 +34,11 @@
 #define SOLCLIENT_CONTEXT_PROP_DEFAULT_MULTI_THREAD   SOLCLIENT_PROP_ENABLE_VAL
 
 #define SOLCLIENT_SESSION_PROP_SSL_PROTOCOL  "SESSION_SSL_PROTOCOL"  /**< This property specifies a comma separated list of the encryption protocol(s) to use. Allowed protocols are 'SSLv3', 'TLSv1', 'TLSv1.1', 'TLSv1.2'. Default: ::SOLCLIENT_SESSION_PROP_DEFAULT_SSL_PROTOCOL. */
-#define SOLCLIENT_SESSION_DEFAULT_PROP_SSL_PROTOCOL          "TLSv1.2,TLSv1.1,TLSv1,SSLv3"         /**< The default value for ::SOLCLIENT_SESSION_PROP_SSL_PROTOCOL*/ 
+#define SOLCLIENT_SESSION_DEFAULT_PROP_SSL_PROTOCOL          ""         /**< The default value for ::SOLCLIENT_SESSION_PROP_SSL_PROTOCOL*/ 
+
```

Comment:
Looks like this is a bit behind GA CCSMP.
I have a script, /home/gszokoli/update_ccsmp_in_go_api.sh that can do the CCSMP update reliably. I just added cache support to it.